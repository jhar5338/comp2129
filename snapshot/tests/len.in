len a
set a 1 2 3 4 5
get a
len a
set b
len b
bye
