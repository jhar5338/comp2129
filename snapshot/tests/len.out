> no such key

> ok

> [1 2 3 4 5]

> 5

> ok

> 0

> bye
